openedx/core/djangoapps/theming/management/commands/setup_edly_multisite_devstack_Juniper_locally.py

```diff
@@ -0,0 +1,259 @@
+                "STUDIO_NAME": "Edly Site",
+                "CSRF_TRUSTED_ORIGINS": [
+                    ".edx.devstack.lms",
+                    "panel.edx.devstack.lms:3030"
+                ],
+                "CORS_ORIGIN_WHITELIST": [
+                    "https://apps.edx.devstack.lms:3030"
+                ],
+                "CMS_BASE": "edx.devstack.lms:18010",
+                "LOGIN_REDIRECT_WHITELIST": [
+                    "edx.devstack.lms:18010"
+                ],
+                "MARKETING_SITE_ROOT": "http://wordpress.edx.devstack.lms/",
+                "LMS_BASE": "edx.devstack.lms:18000",
+                "FEATURES": {
+                    "PREVIEW_LMS_BASE": "edx.devstack.lms:18000"
+                },
+                "MKTG_URLS": {
+                    "ROOT": "http://wordpress.edx.devstack.lms/",
+                    "NAV_MENU": "wp-json/edly-wp-routes/nav-menu",
+                    "FOOTER": "wp-json/edly-wp-routes/footer"
+                },
+                "LMS_SEGMENT_KEY": "",
+                "BADGR_ISSUER_SLUG": "",
+                "CMS_SEGMENT_KEY": "",
+                "BADGR_USERNAME": "",
+                "BADGR_PASSWORD": ""
+            }
+        })
+
+        organization, _ = Organization.objects.get_or_create(name='Edly', short_name='edly')
+        edly_organization, _ = EdlyOrganization.objects.get_or_create(name='Edly Organization', slug='edly',
+                                                                      enable_all_edly_sub_org_login=True)
+
+        edly_suborganization, _ = EdlySubOrganization.objects.get_or_create(
+            name='Edly Organization', slug='edly',
+            edly_organization=edly_organization,
+            edx_organization=organization,
+            lms_site=lms_site, studio_site=studio_site,
+            preview_site=lms_site
+        )
+
+        edly_suborganization.edx_organizations.set([organization])
+
+        Switch.objects.get_or_create(name='DISABLE_ACCOUNT_ACTIVATION_REQUIREMENT_SWITCH', active=True)
+        Switch.objects.get_or_create(name='ENABLE_COURSE_PROGRESS_SWICTH', active=True)
+        Switch.objects.get_or_create(name='enable_edly_organizations', active=True)
+        Switch.objects.get_or_create(name='completion.enable_completion_tracking', active=True)
+        Switch.objects.get_or_create(name='ENABLE_NOTIFICATIONS', active=True)
+        Switch.objects.get_or_create(name='verify_student_disable_account_activation_requirement', active=True)
+        Switch.objects.get_or_create(name='enable_course_progress_feature', active=True)
+
+    @staticmethod
+    def update_credentials():
+        """Update credentials and redirect uris."""
+        Application.objects.filter(name='credentials-sso').update(
+            redirect_uris='http://edx.devstack.lms:18150/complete/edx-oauth2/')
+
+        CatalogIntegration.objects.get_or_create(internal_api_url='https://edx.devstack.lms:1381/api', enabled=True)
```

Comment:
can you confirm the port number ? isn't it `18381` ?

Reply:
According to the Confluence document, the port is 1381.

Reply:
There must be typo in confluence, please fix it there as well.

Reply:
Done